File: spring-boot-sample/velocity/src/main/java/cn/tim/velocity/entity/Event.java
package cn.tim.velocity.entity;

import lombok.Data;

import java.io.Serializable;
import java.util.Arrays;
import java.util.Date;
import java.util.Random;

/**
 * Created by luolibing on 2017/4/6.
 */
@Data
public class Event implements Serializable {
    private static Random rand = new Random(47);
    private static long counter = 0;
    private Long id = counter++;
    private String eventType = EventType.randEventType().eventDesc;
    private String title = "操作";
    private String content = rand.nextBoolean() ? "操作内容" : null;
    private Date createDate = new Date();

    public int size(String[] array) {
        return array.length;
    }

    public void execute() {
        System.out.println("execute " + toString());
    }

    public void setArray(String...args) {
        System.out.println("array = " + Arrays.toString(args));
    }
}

File: modules/vue/src/main/resources/static/js/event.js
/**
 * Created by luolibing on 2018/2/8.
 */

var app1 = new Vue({
    el: '#app1',
    data: {
        counter: 0
    },
    methods: {
        greet: function(event) {
            alert(this.counter);
            if(event) {
                alert(event.target.tagName)
            }
        }
    }
});